static/js/tools/map/chart_loader.tsx

```diff
@@ -169,15 +169,21 @@ function loadChartData(
       continue;
     }
     let statVarValue = statVarData.stat[dcid].value;
-    if (isPerCapita && dcid in populationData) {
-      const popDate = getPopulationDate(
-        populationData[dcid],
-        statVarData.stat[dcid]
-      );
-      statVarValue = statVarValue / populationData[dcid].data[popDate];
-      sources.add(populationData[dcid].provenanceUrl);
-    } else if (isPerCapita && !(dcid in populationData)) {
-      continue;
+    if (isPerCapita) {
+      if (dcid in populationData) {
+        const popDate = getPopulationDate(
+          populationData[dcid],
+          statVarData.stat[dcid]
+        );
+        const popValue = populationData[dcid].data[popDate];
+        statVarValue =
+          popValue === 0
+            ? statVarValue
```

Comment:
sorry, on second thought - population = 0 should be treated the same way. they're both invalid values.

so we could display it the same as "data missing", but perhaps in the tooltip, we should also say 'missing population data'.

we might want to differentiate between these "data missing" and "data invalid" states as well - ok to mark this as a todo for now.

Reply:
done. For now, the tooltip content can't be customized to say "missing population data", but that is the next thing I will be working on (tooltip content to be customizable). And added a todo for differentiating between "data missing" and "data invalid".